samples/nrf9160/modem_shell/src/cloud/cloud_shell.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef CLOUD_SHELL_H
+#define CLOUD_SHELL_H
+
+int cloud_shell(const struct shell *shell, size_t argc, char **argv);
```

Comment:
Is this file needed somewhere? At least this function doesn't exist anymore.
'gnss' module seems to have similar issue.

Reply:
Good catch, I'll remove cloud_shell.h. I can do similar change for GNSS as well in this PR but in a separate commit. I hope it's ok.

Reply:
removed